masonry/src/lib.rs

```diff
@@ -1,6 +1,9 @@
 // Copyright 2018 the Xilem Authors and the Druid Authors
 // SPDX-License-Identifier: Apache-2.0
 
+// cargo rdme --workspace-project=masonry
+// After editing the below, then check links in README.md
```

Comment:
This comment is a bit unclear, and there's a typo in the second line. Same for all the other files with the same comment.

Reply:
"After editing the below" should probably have been "After editing the doc below".

In any case, current text is fine.